src/preprocessor.ts

```diff
@@ -60,7 +60,7 @@ export function process(
       const outputFilePath = nodepath.join(
         config.cacheDirectory,
         '/ts-jest/',
-        new Buffer(path).toString('base64'),
+        crypto.createHash('md5').update(path).digest('hex'),
       );
```

Comment:
With this change, we no longer need to shorten the path as it is being done in the prior statement. Can you remove that and the associated comment?